astrology.lic

```diff
@@ -286,12 +286,17 @@ class Astrology
   end
 
   def train_astrology(settings)
+    return unless DRStats.moon_mage?
     return unless settings
-    return unless settings.astrology_training
+    return unless settings.astrology_training.none?
```

Comment:
This will throw if `settings.astrology_training` is `nil` (`undefined method `none?' for nil:NilClass (NoMethodError)`).